Resources/Prototypes/Catalog/VendingMachines/Inventories/theater.yml

```diff
@@ -41,6 +41,7 @@
     ClothingUniformJumpsuitKimono: 1
     ClothingHeadHatCasa: 1
     ClothingHeadHatHairflower: 1
+    ClothingShoesBootsFisher : 1
```

Comment:
Does this need to be here? None of the other chef gear is here.

Reply:
I additionally put it there because I felt it fit in with the other silly items from there, but now that I'm properly considering it again, it feels both more consistent and funnier to just have it in the chef vend. Changing this in the next commit.